sep.pyx

```diff
@@ -838,6 +857,16 @@ def sum_circle(np.ndarray data not None, x, y, r,
     maskthresh : float, optional
         Threshold for a pixel to be masked. Default is ``0.0``.
 
+    seg : `numpy.ndarray`, optional
+        Segmentation image with dimensions of `data` and dtype `np.float32`.
```

Comment:
dtype `np.int32`?

Reply:
Agreed.  I just renamed "seg" to "segmap" throughout and updated the docstrings to make the direct connection to the output from `sep.extract` more clear.